src/auto-evo/RunResults.cs

```diff
@@ -389,27 +387,26 @@ public void PrintSummary(PatchMap previousPopulations = null)
         /// <param name="playerReadable">if true ids are removed from the output</param>
         /// <param name="effects">if not null these effects are applied to the population numbers</param>
         /// <returns>The generated summary text</returns>
-        public string MakeSummary(PatchMap previousPopulations = null,
+        public LocalizedStringBuilder MakeSummary(PatchMap previousPopulations = null,
             bool playerReadable = false, List<ExternalEffect> effects = null)
         {
             const bool resolveMigrations = true;
             const bool resolveSplits = true;
 
-            var builder = new StringBuilder(500);
+            var builder = new LocalizedStringBuilder(500);
 
-            string PatchString(Patch patch)
+            LocalizedStringBuilder PatchString(Patch patch)
```

Comment:
I think this should be refactored so that this is given in the LocalizedStringBuilder where the patch data is added to. This would reduce this nested LocalizedStringBuild stuff, which I kind of dislike as to me it seems we might end up with very deep object hierarchies unnecessarily.

Reply:
I dislike this too, but this stringbuilder is used in another formatString
```csharp
builder.Append(new LocalizedString("WENT_EXTINCT_IN", patchName));
```
So this nesting is needed

Reply:
I guess it can't be avoided, then. At least a good design that takes that into account and avoid the nesting isn't immediately popping into my mind.